package.json

```diff
@@ -14,6 +14,7 @@
   "devDependencies": {
     "globby": "^13.2.2",
     "husky": "^8.0.3",
+    "jest": "^29.7.0",
```

Comment:
We don't use Jest anymore, we use Vitest now.

Reply:
Ok. Got it